basicsr/archs/hifacegan_arch.py

```diff
@@ -0,0 +1,284 @@
+'''
+  HiFaceGAN_arch: Network definition of HiFaceGAN
+'''
+import numpy as np
```

Comment:
Just a convention in basicsr:
1. we do not use comments at the head of a file.  And we just put it in the docstring of the corresponding class.

Reply:
Agreed